### DISCUSSION TITLE
[A-L] (2023/24) Foglio 7 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#315] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-11-12 150741](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/cb873029-c12d-4935-9c97-9c58b60e9fcc)

**Comment by CiottoloMaggico** [comment]
Soluzione alternativa
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/db00d446-c0e7-463d-90b4-9031dcda07ac)

**Comment by daromacarlo** [comment]
![Immagine 2024-01-18 171216](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/916006fc-bf2d-4155-ba42-166fc6e251f4)
![Immagine 2024-01-18 171236](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/4ccad073-99a3-4d71-8318-407f2f4679ec)
![Immagine 2024-01-18 172416](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/00c24f58-9f55-45bd-9369-1277f2bb4b13)
![Immagine 2024-01-18 172426](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/04f08a4e-8a59-4483-9c37-ebc6138bdbad)
